pkgs/development/python-modules/pydantic-settings/default.nix

```
@@ -50,11 +50,20 @@ buildPythonPackage rec {
     export HOME=$TMPDIR
   '';
 
+  # ruff is a transiant dependency of pytest-examples which is required to run the tests.
```

```suggestion
  # ruff is a dependency of pytest-examples which is required to run the tests.
```
Pytest-examples uses ruff at runtime. Or what do you mean by transient?

---

Done !